ydlidar_ros2_driver_node:
  ros__parameters:
    port: /dev/ttyUSB0
    frame_id: base_laser_back_link
    sensor_name: lidar_back
    lidar_topic_name: /scan_raw_back
    pointcloud_topic_name: /raw_pointcloud_back
    publisher_qos: "reliable" # "best_effort" or "reliable"
    ignore_array: ""
    baudrate: 512000
    lidar_type: 0
    device_type: 0
    sample_rate: 20
    abnormal_check_count: 4
    fixed_resolution: true
    reversion: true
    inverted: true
    auto_reconnect: true
    isSingleChannel: false
    intensity: false
    support_motor_dtr: false
    angle_max: 127.5
    angle_min: -127.5
    range_max: 15.0
    range_min: 0.01
    frequency: 10.0
    invalid_range_is_inf: false
